apps/src/StudioApp.js

```diff
@@ -1811,6 +1811,25 @@ StudioApp.prototype.handleEditCode_ = function (config) {
   var aceEditor = this.editor.aceEditor;
   aceEditor.session.setMode('ace/mode/javascript_codeorg');
 
+  // Extend the command list on the ace Autocomplete object to include the period:
+  var Autocomplete = window.ace.require("ace/autocomplete").Autocomplete;
+  Autocomplete.prototype.commands['.'] = function(editor) {
```

Comment:
Okay, I think I get it.  So `exactMatch` is an exact match so far in the completions list - if I've typed `Math.` or `console.` then the examples you game above are an exact match and the completer stays open with those options showing. Otherwise it closes after the dot, and re-opens using the new method completer after you type the next character.  Is that correct?